app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -174,16 +160,18 @@ public function indexAction()
      */
     public function prefetchEstimateAction()
     {
-
         set_time_limit(30);
         ignore_user_abort(true);
 
         /** @var Mage_Sales_Model_Quote $quote */
         $quote = Mage::getSingleton('checkout/session')->getQuote();
 
         if(!$quote->getId() || !$quote->getItemsCount()){
-            $this->getResponse()->clearAllHeaders()->setHeader('Content-type', 'application/json');
-            $this->getResponse()->setBody("{}");
+            $this->sendResponse(
```

Comment:
not directly related to this PR but shouldn't this be error?

Reply:
It is related and included because `sendResponse` is an ApiControllerTrait method.